utils/gyb.py

```diff
@@ -716,7 +716,8 @@ def execute(self, context):
 
         # If we got a result, the code was an expression, so append
         # its value
-        if result is not None and result != '':
+        if result is not None \
+                or (isinstance(result, basestring) and result != ''):
```

Comment:
I'm pretty sure gyb would require a lot of changes to work with python3, but do you have a preferred solution?

http://stackoverflow.com/questions/11301138/how-to-check-if-variable-is-string-with-python-2-and-3-compatibility